helm/aws-admission-controller/templates/psp.yaml

```diff
@@ -35,3 +36,4 @@ spec:
   hostNetwork: false
   hostIPC: false
   hostPID: false
+{{- end }}
```

Comment:
Please, [add new line at the end](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).